server/pom.xml

```diff
@@ -203,6 +203,13 @@
                   <pattern>com.fasterxml.jackson.databind</pattern>
                   <shadedPattern>${rss.shade.packageName}.jackson.databind</shadedPattern>
                 </relocation>
+                <relocation>
```

Comment:
Could you revert this change?

Reply:
Hi @jerqi Thanks for the kind reminder and suggestions. I updated the part you mentioned.